src/serve-panel-provider.ts

```diff
@@ -99,6 +99,22 @@ export class ServePanelProvider implements vscode.WebviewViewProvider {
           break;
         }
 
+        case 'sudoPrompt': {
+          Logger.info('running tsrelay in sudo');
+          try {
+            await this.ts.initSudo();
+            Logger.info(`re-applying ${m.operation}`);
+            if (m.operation == 'add') {
+              await this.ts.serveAdd(m.params!);
```

Comment:
We should avoid the non-null assertion to avoid possible issues, as well as it's currently generating a Typescript warning (`yarn tsc --noEmit`). We can check for params when we check the operation.

Reply:
Will do as an early return in an if-check, feel free to change it further 👍🏼